admin/tabs/general/about/AboutPanel.js

```diff
@@ -41,30 +43,33 @@ function renderTables() {
     return [
         table({
             item: tbody(
-                row('App Name / Code', `${svc.get('appName')} / ${svc.get('appCode')}`),
-                row('Environment', svc.get('appEnvironment')),
-                row('Database', svc.get('databaseConnectionString')),
-                row('DB User / Create Mode', `${svc.get('databaseUser')} / ${svc.get('databaseCreateMode')}`),
+                row('App Name / Code', `${get('appName')} / ${get('appCode')}`),
+                row('Environment', get('appEnvironment')),
+                row('Database', get('databaseConnectionString')),
+                row('DB User / Create Mode', `${get('databaseUser')} / ${get('databaseCreateMode')}`),
+                row('App Timezone', fmtTimezone(get('appTimezone'), get('appTimezoneOffset'))),
+                row('Server Timezone', fmtTimezone(get('serverTimezone'), get('serverTimezoneOffset'))),
+                row('Client Timezone',  fmtTimezone('Unknown', (new Date()).getTimezoneOffset()*MINUTES*-1)),
```

Comment:
Note we could use `Intl.DateTimeFormat().resolvedOptions().timeZone` if we wanted the display the "Americas/Los Angeles" form of the client TZ alongside the more descriptive zone names for server/app.

(Also note the capital Z in that API.)

Reply:
went with the capital Z on server and client -- think that makes sense(although our current Clock component is using the small Z as the JavaScript Date API....)

Fortunately, moment uses `tz` escaping the issue largely!